### PR TITLE
Swup dev2

### PR DESCRIPTION
**Pull request by mozoko**

### CODE REVIEW

File: recipes-debian/fontforge/fontforge_debian.bb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    # uthash sources are expected in uthash/src
+    currdir=`pwd`
+    cd ${S}
+
+    mkdir -p uthash/src
+    cp ${STAGING_INCDIR}/ut*.h uthash/src
+
+    # avoid bootstrap cloning gnulib on every configure
+    cat >.gitmodules <<EOF
+[submodule "gnulib"]
```
Comment (masami256) [inline recipes-debian/fontforge/fontforge_debian.bb:37]: I think this recipe shouldn't  get gnulib from git, use debian package instead.

File: recipes-debian/giflib/giflib_debian.bb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# base recipe: meta-oe/recipes-devtools/giflib/giflib_5.1.4.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+SUMMARY = "shared library for GIF images"
+SECTION = "libs"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://COPYING;md5=ae11c61b04b2917be39b11f78d71519a"
+
+SRC_URI = "${SOURCEFORGE_MIRROR}/giflib/${BP}.tar.bz2"
```
Comment (masami256) [inline recipes-debian/giflib/giflib_debian.bb:10]: Do not need it.

File: recipes-debian/libfont-ttf-perl/libfont-ttf-perl_debian.bb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Perl module for TrueType font hacking"
```
Comment (masami256) [inline recipes-debian/libfont-ttf-perl/libfont-ttf-perl_debian.bb:1]: please add headers(e.g. base recipe).

File: recipes-debian/libio-string-perl/libio-string-perl_debian.bb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Emulate IO::File interface for in-core strings"
```
Comment (masami256) [inline recipes-debian/libio-string-perl/libio-string-perl_debian.bb:1]: please add headers(e.g. base recipe).

**Comment by mozoko** [comment]
Test is not done sufficiently, so I'll close this PR.

**Pull request closed without merging**